USING-CONFLUENCE.md

```diff
@@ -0,0 +1,69 @@
+# Using Confluence
+
+The confluence command uses the [Web API Confluence Dashboard](http://web-confluence.appspot.com/#!/) to generate new or improved browser compatibility data ready for submission in pull requests. The Web API Confluence Dashboard provides metrics and support data about web platform features across multiple browsers. The MDN Browser Compatibility Database provides a command for using Confluence to generate new and updated JSON files. This page explains how to do this.
+
+## Prerequisites
+
+Before using the confluence command make sure you have done all of the items below.
+
+* On GitHub fork this repo.
+* In this repo, add your fork as a remote repo: `git remote add fork-name path-to-fork`
+* In the root of your repo clone run `npm install`.
+* Configure a git [difftool](https://git-scm.com/docs/git-difftool).
+
+## Basic Procedure
+
+1. On your local `master` branch run `git pull` to update your local data.
+1. Create a new branch using `git checkout -b my-new-branch-name`.
+1. Call the confluence command, passing in the name of web platform interface. For example:
+
+`npm run confluence -- --interfaces=ServiceWorker`
+
+If the confluence command finds any differences between existing browser compatibility data and Web API Confluence, it will write a new file. Confirm that it did so by running `git status`.
+1. Open the file that changed and manually copy values from `chrome` to `chrome_android` and `webview_android`. See [Only Desktop Data is Available](#only-desktop-data-is-available) for an explanation. Save the file.
+1. To review changes made to a JSON file, run `git difftool`. Save the file.
+1. Commit your changes using `git add .` then `git commit`.
+1. Push the branch containing your changes to your repo fork: `git push --set-upstream remote-name branch-name`.
```

Comment:
If we stick with the clone suggestion in the prereqs section, otherwise ignore the remote bit
```suggestion
1. Push the branch containing your changes to your fork: `git push --set-upstream origin branch-name`.
```